cpp/src/parquet/arrow/writer.cc

```diff
@@ -469,6 +415,30 @@ class FileWriterImpl : public FileWriter {
     return writer_->metadata();
   }
 
+  Status WriteTableUnbuffered(const Table& table, int64_t chunk_size) {
```

Comment:
```suggestion
  Status WriteNonBufferedRowGroup(const Table& table, int64_t chunk_size) {
```

Reply:
Also, this more write more than one rowgroup(at least a row-group is written), so I guess `WriteTable` is ok?